frontend/src/app/components/wp-card-view/wp-single-card/wp-single-card.component.sass

```diff
@@ -40,10 +40,13 @@
   grid-template-columns: auto 1fr auto
   grid-template-rows: auto auto 1fr auto
```

Comment:
You added a new line in `grid-template-areas` but forgot to define it here. Right now, this works because CSS-grid is quite smart and simply calculates the additional row itself. However, we should still add it here.